protos/flyteidl/service/admin.proto

```diff
@@ -155,13 +187,22 @@ service AdminService {
         get: "/api/v1/launch_plans/{id.project}/{id.domain}"
       }
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "Fetch existing launch plan definitions matching input filters."
+    };
   }
 
   rpc UpdateLaunchPlan (flyteidl.admin.LaunchPlanUpdateRequest) returns (flyteidl.admin.LaunchPlanUpdateResponse) {
     option (google.api.http) = {
       put: "/api/v1/launch_plans/{id.project}/{id.domain}/{id.name}/{id.version}"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "Update the status of an existing launch plan definition. "
+      "At most one launch plan version for a given {project, domain, name} can be active at a time. "
+      "If this call sets a launch plan to active and existing version is already active, the result of this call will be that the "
+      "formerly active launch plan will be made inactive and specified launch plan in this request will be made active."
```

Comment:
We should mention that if the launch plan is associated with a schedule, then the schedule will be activated while the existing schedule will be deactivated.

Reply:
ok, done